### PR TITLE
[Solution#172] raises a Mix.Error instead of choking on invalid requi…

### PR DESCRIPTION
**Pull request by RobertDober**
…rements (only fot hex requirements)

Seems easy enough after all, thx to @lexmag for clearing things up.

### CODE REVIEW

File: lib/hex/remote_converger.ex [hunk 1]
```diff
@@ -105,6 +105,9 @@ defmodule Hex.RemoteConverger do
   end
 
   defp check_package_req(name, req, from) do
+    if req && Version.parse_requirement(req) == :error do
```
Comment (lexmag) [inline lib/hex/remote_converger.ex]: `nil` requirement is still valid.

**Comment by lexmag** [comment]
> What would be a good commit message for PRs. I guess I shall have a look at some other PRs.

Regarding commit message, I think this guide is pretty good: https://github.com/erlang/otp/wiki/Writing-good-commit-messages.

File: lib/hex/remote_converger.ex (hunk 1 above)
Comment (RobertDober) [inline lib/hex/remote_converger.ex]: That is why I do not raise a Mix.Error in that case.

**Comment by lexmag** [comment]
Thank you. :yellow_heart:

**Comment by RobertDober** [comment]
And thanx to you too, Happy Xmas. :christmas_tree:

**Pull request merged**